### ISSUE TITLE
Application does not appear after scaling to 0 instances and back to 1+

### ISSUE THREAD

**Issue by thehejik** [epinio/epinio#2161]
**Describe the bug**
After scaling an existing app to 0 instances and then scaling back to a positive amount of instances the app won't re-appear.

Visible in `epinio:v1.7.1`

**To Reproduce**
```
$ epinio app update sample --instances 0
$ epinio app list

🚢  Listing applications
Namespace: workspace

✔️  Epinio Applications:
|  NAME  |            CREATED            | STATUS | ROUTES | CONFIGURATIONS | STATUS DETAILS |
|--------|-------------------------------|--------|--------|----------------|----------------|
| sample | 2023-03-30 11:18:49 +0000 UTC | n/a    | n/a    |                |                |
$ epinio app update sample --instances 3
$ epinio app list

🚢  Listing applications
Namespace: workspace

✔️  Epinio Applications:
|  NAME  |            CREATED            | STATUS | ROUTES | CONFIGURATIONS | STATUS DETAILS |
|--------|-------------------------------|--------|--------|----------------|----------------|
| sample | 2023-03-30 11:18:49 +0000 UTC | n/a    | n/a    |                |                |
$ kubectl get pods -n workspace
No resources found in workspace namespace.
```

**Expected behavior**
the workload should re-appear with given amount of instances

**Comment by gitricko** [comment]
This seems to be visible also in 1.5.2

**Comment by andreas-kupries** [comment]
Looking at server logging the client makes patch request when upating an application, like the number of desired instances. This patch request triggers an update of the app helm chart. Except when trying to scale the app from zero instances. There the server receives the patch request and no chart update looks to be triggered.

Investigating that part of the API now.

**Comment by thehejik** [comment]
Verified fixed in `v1.8.0-rc1` (see PR)
